furusiya/model/maps/generators/forest_generator.py

```diff
@@ -134,6 +134,12 @@ def _random_walk(self, num_tiles):
     def place_stairs(self):
```

Comment:
This should be easily testable. It doesn't rely on anything other than `config`, and I figured out how to mock/setup `config` in tests.